model/Core/Vocabularies/DocumentState.md

```diff
@@ -0,0 +1,23 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DocumentState
+
+## Summary
+
+The state of a document is defined by the enumeration.
+
+## Description
+
+Documents states vary by state. Enumerations are used to define the state.
+
+## Metadata
+
+- name: DocumentState
+
+## Entries
+
+- Draft: in production 
+- Active: in use
+- Obsolete: superseded or not valid at present
+- Other: other
```

Comment:
```suggestion
- draft: in production 
- active: in use
- obsolete: superseded or not valid at present
- other: other
```

Not sure if entries need to start with lowercase or not?
@goneall @zvr @kestewart  Do we have a convention on this?

Looks like entries in other vocabs are start with lowercase:

- https://github.com/spdx/spdx-3-model/blob/develop/model/Security/Vocabularies/SsvcDecisionType.md
- https://github.com/spdx/spdx-3-model/blob/develop/model/Core/Vocabularies/HashAlgorithm.md
- https://github.com/spdx/spdx-3-model/blob/develop/model/Core/Vocabularies/RelationshipCompleteness.md
- https://github.com/spdx/spdx-3-model/blob/develop/model/Software/Vocabularies/SoftwarePurpose.md